packages/components/src/__stories__/WIP/Presets.stories.js

```diff
@@ -1,34 +1,31 @@
 import { ui } from '@wp-g2/styles';
 import React from 'react';
 
-import { HStack, Spacer, View } from '../../index';
+import { HStack, View } from '../../index';
 
 export default {
 	title: 'Examples/WIP/Presets',
 };
 
-const Thing = ({ children = 'View', ...props }) => (
-	<View
-		{...props}
-		css={[
-			ui.font.default,
-			ui.background.green,
-			ui.padding(3),
-			ui.borderRadius.round,
-			ui.border.all,
-			ui.animation.ease,
-			ui.hover(ui.background.red, ui.scale(1.2)),
-			ui.mode.dark([`border: 3px solid white`]),
-			ui.$('NestedThing').css([`border: 2px solid black`]),
-		]}
-	>
-		{children}
-	</View>
-);
-
 const Tammie = () => {
 	return (
-		<View css={[ui.scale(1), ui.hover(ui.scale(2)), ui.animation.default]}>
+		<View
+			css={[
+				ui.padding(5),
+				ui.background.blue,
+				ui.scale(1),
+				ui.hover(ui.scale(1.5), ui.background.orange),
+				ui.animation.default,
+				{
+					boxShadow: ui.flow(
+						ui.calc(ui.get('controlHeight'), '+', ui.value.px(40)),
+						ui.get('gridBase'),
+						ui.calc('calc(1px + 20px)', '+', 'calc(1px + 20px)'),
```

Comment:
Shouldn't this be `ui.flow.calc`? Where is `ui.calc` being set?

Reply:
Oh! Thank you 🙏 . My initial attempt at `flow` did indeed have `calc` at the main `ui` level. I changed it afterwards.